core/constants.py

```diff
@@ -18,13 +18,23 @@
 
 from __future__ import annotations
 
+import io
 import json
+import os
 import re
-
-from core import python_utils
+import sys
 
 from typing import Any, Dict
 
+_THIRD_PARTY_PATH = os.path.join(os.getcwd(), 'third_party', 'python_libs')
+sys.path.insert(0, _THIRD_PARTY_PATH)
```

Comment:
Please add a comment explaining why is this needed.

Reply:
Done

Reply:
I think you will need to elaborate more in your comment on why exactly is this needed.

Reply:
yes I figure if I move it to the exact place it requires, its easier to explain so i moved it to `feconf.py`